earthfile2llb/interpreter.go

```diff
@@ -493,6 +497,10 @@ func (i *Interpreter) handleRun(ctx context.Context, cmd spec.Command) error {
 		return nil
 	}
 
+	if i.target.IsRemote() && *privileged {
+		return i.errorf(cmd.SourceLocation, "Permission denied: unwilling to run privileged command from remote Earthfile")
+	}
+
```

Comment:
This looks good, but I wouldn't make this change right now as it might break people's builds and have no way to get them back. First we need a setting to allow this behavior: "i know what I'm doing" kind of thing.

Perhaps we can do this in a later PR and only focus on `LOCALLY` for now? `LOCALLY` should be fine to change in this PR because it's an experimental feature.. so we made no promises to maintain API / compatiblity.

Reply:
I should have left this as a Draft PR; I'm happy to leave it open until we have a complete story for listing trusted repos

Reply:
Sounds good